src/Stripe.net/Stripe.net.csproj

```diff
@@ -54,6 +54,10 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="7.0.0" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="System.Configuration.ConfigurationManager" Version="8.0.0" />
+  </ItemGroup>
```

Comment:
Is it safe to add this dependency?

Reply:
I think we can add to https://github.com/stripe/stripe-dotnet/blob/master/src/StripeTests/StripeTests.csproj

Reply:
I think this should have already existed, looking at https://github.com/stripe/stripe-dotnet/pull/2650/files? Users can set this as well outside of tests no?

Reply:
Oh OK I missed that